app/src/main/java/org/openobservatory/ooniprobe/activity/MeasurementDetailActivity.java

```diff
@@ -54,19 +48,17 @@
 import org.openobservatory.ooniprobe.test.test.WebConnectivity;
 import org.openobservatory.ooniprobe.test.test.Whatsapp;
 
-import java.io.File;
 import java.io.Serializable;
-import java.nio.charset.Charset;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 import localhost.toolkit.app.fragment.ConfirmDialogFragment;
-import localhost.toolkit.app.fragment.MessageDialogFragment;
-import okhttp3.Request;
 
 public class MeasurementDetailActivity extends AbstractActivity implements ConfirmDialogFragment.OnConfirmedListener {
     private static final String ID = "id";
+    private static final int TYPE_LOG = 1;
+    private static final int TYPE_JSON = 2;
```

Comment:
Is it possible to avoid defining these definitions twice? Maybe a package specific Java enum?